omod/src/main/java/org/openmrs/module/fhir2/web/servlet/Fhir2SmartConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.web.servlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.openmrs.module.fhir2.web.SmartConformance;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Fhir2SmartConfig extends HttpServlet {
+	
+	public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+		
+		ObjectMapper objectMapper = new ObjectMapper();
+		
+		SmartConformance smartConformance = new SmartConformance();
```

Comment:
Any reason these can't be class objects? I.e., so we only have one `objectMapper` and one `smartConformance` per servlet, rather than creating new ones for every request.

Reply:
@ibacher Not sure about what should be done here.

Reply:
@ibacher Not sure about what should be done here.

Reply:
Oh... I just meant turn them into class properties:

```java

class Fhir2SmartConfig extends HttpServlet {
    private static final ObjectMapper objectMapper = new ObjectManager();

    private final SmartConformance smartConformance;

    public Fhir2SmartConfig() {
        this.smartConformance = new SmartConformance();
        // set properties here
    }

    public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
        PrintWriter out = res.getWriter();
		res.setContentType("application/json");
		res.setCharacterEncoding("UTF-8");
		res.setStatus(200);
		objectMapper.writeValue(out, smartConformance);
        // why do we need these next two lines?
		out.print(objectMapper);
		out.flush();
    }
}
```